content/zh-cn/includes/task-tutorial-prereqs-node-upgrade.md

```diff
@@ -0,0 +1,7 @@
+<!--
+You need to have shell access to all the nodes, and the kubectl command-line tool must
+be configured to communicate with your cluster. It is recommended to run this tutorial 
+on a cluster with at least two nodes that are not acting as control plane hosts.
+-->
+你必须拥有一个 Kubernetes 的集群，同时你必须配置 kubectl 命令行工具与你的集群通信。
```

Comment:
```suggestion
你必须有 Shell 能访问所有节点，且必须配置 kubectl 命令行工具让其与你的集群通信。
```

Reply:
This is the same translation as `content/zh-cn/includes/task-tutorial-prereqs.md`.

Do you think it needs to be changed together?

Reply:
Seems yes, the translation has some gap to the original english.

Reply:
Oh, I misread it. The two files are different.